### PR TITLE
feature: new method to send card cvn during authentication

### PR DESCRIPTION
**Pull request by javiersuweijie**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,7 +31,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
         debug {
-            minifyEnabled true
+            minifyEnabled false
```
Comment (adhiartaputra) [inline app/build.gradle:34]: why we change it from true to false? @javiersuweijie

File: app/src/main/java/com/xendit/example/AuthenticationActivity.java [hunk 2]
```diff
@@ -73,8 +73,9 @@ public void onClick(View view) {
 
         String tokenId = tokenIdEditText.getText().toString();
         String amount = amountEditText.getText().toString();
+        String cardCvn = cardCvnEditText.getText().toString();
```
Comment (adhiartaputra) [inline app/src/main/java/com/xendit/example/AuthenticationActivity.java:76]: Do you think we should add currency here too? @javiersuweijie

**Review by adhiartaputra — COMMENTED** [review]
small comment otherwise LGTM

File: app/build.gradle (hunk 1 above)
Comment (javiersuweijie) [reply app/build.gradle:34]: cant debug properly if we minify it during debug

**Pull request merged**